initializers/sequelize.js

```diff
@@ -99,3 +118,62 @@ module.exports = {
     });
   }
 };
+
+function migrateSequelizeMeta(api, umzug) {
```

Comment:
If this function updates the old migration tables for you, is the warning in the README necessary?